### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by haraphat01**

### CODE REVIEW

**Comment by haraphat01** [comment]
This is final work for my Newsweek project, built with HTML, CSS, and Boostrap. Kindly review it for your kind approval.

**Comment by trekab** [comment]
Hello @haraphat01, it is unfortunate the I have to mark your code review request because your pull request lacks a meaningful name and a short summary.

Image PR example

Read more about this from this [guide](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
_Feel free to slack me for any inquiries_
__Treasure Kabareebe__

**Comment by haraphat01** [comment]
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices. 
The technology used was basically;

- Html

- CSS and,

- Bootstrap.



The final project is what I'm submitting now to the TSE for review.

**Comment by trekab** [comment]
# Changes Required!
Hi @haraphat01, I'd like to start by commending you on a great job done on this project so far and trying your best to keep your repo professional. :+1:  There are some changes I'd like you to look at though. These include;

- [ ] Your navigation bar breaks on small screens. ([clone screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/clone_sm_header.png), [orignal site screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/original_sm_nav.png))
- [ ] Top Story should be on top for small screens in the main section. ([clone screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/clone_sm_header.png), [orignal site screenshot](https://raw.githubusercontent.com/trekab/images/master/arafat_akata/original_sm_nav.png))
- [ ] Your more stories section([screenshot](https://github.com/trekab/images/blob/master/arafat_akata/more_stories_clone.png?raw=true)) breaks on small screen. (compare [original](https://github.com/trekab/images/blob/master/arafat_akata/more_stories_original.png?raw=true))

__Please work on the above changes and submit another code review request__
_Feel free to contact me on slack for any inquiries about this review_ :wink: 
__Treasure Kabareebe__

**Comment by haraphat01** [comment]
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

- I fixed the broken nav-bar
- The Top Story Section is now displaying at the top

- The more stories section has also been fixed

**Comment by sumancrest0001** [comment]
**Nice work @haraphat01 :clap: :clap:**
Breaking points are working correctly but we need to fix a couple of issues related to alignment, styling, and placement of elements to get a green signal from our side. Since it is a clone project, we have to try to achieve a similar design. Typography is essential and let's try to use similar fonts, font-sizes, and font-weights. Kindly make the following changes and submit them for another review. 
**[NOTE] All the requested changes are mandatory.**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,699 @@
...
+.world a {
+  text-decoration: none;
+  color: #f72210;
+}
+
+.world2 a {
+  text-decoration: none;
+  color: #f72210;
+}
+
```
Comment (sumancrest0001) [inline style.css]: Let's avoid repetition of code and follow the DRY principle. you can do something like this `.world a, .world2 a, .newsweek a {
  text-decoration: none;
  color: #f72210;
}`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,699 @@
...
+.anchor2 a:hover {
+  color: red;
+}
+
+.anchor3 {
+  color: black;
+  font-size: 0.9rem;
+}
```
Comment (sumancrest0001) [inline style.css]: CSS styling is repeated for anchor2 and anchor3. You can read [this](https://speckyboy.com/good-bad-css-practices/) article as a reference. [Pathright requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png)

File: style.css [hunk 3]
```diff
@@ -0,0 +1,699 @@
...
+  top: -0.5em;
+}
+
+.debate-picture2 {
+  width: 50px;
+  height: 50px;
+  position: absolute;
+  right: 0;
+  top: 0.5em;
+}
```
Comment (sumancrest0001) [inline style.css:230]: Remove all the repeated codes.

**Comment by sumancrest0001** [comment]
Signup section after the latest news is missing in your project. Please have a look 
in your project :point_down: 
Screenshot from 2020-05-09 11-38-47
It should be :point_down: 
Screenshot from 2020-05-09 11-36-13

**Comment by sumancrest0001** [comment]
Try to fix alignment, margin, padding of the newsletter, subscribe, and footer section on a small screen. It looks congested.
original project :point_down: 
Screenshot from 2020-05-09 11-42-18

in your project :point_down: 
Screenshot from 2020-05-09 11-41-48

**Comment by sumancrest0001** [comment]
Let's remove the second navbar on a medium screen to make it similar to that of the original one. 
currently in your project :point_down: 
Screenshot from 2020-05-09 11-46-48

it should be :point_down: 
Screenshot from 2020-05-09 11-46-57

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Please make the requested changes and request for another review.**
If you need any help or have a question, feel free to DM me on Slack (**Suman Shrestha**)
**Happy Coding !!!**

**Comment by haraphat01** [comment]
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

-  Removed Repeated Codes

- Hid the nav bar on medium screen

- Added the sign up section

- added the font family

- Worked on the footer too

I

**Comment by lekegitrepo** [comment]
#### @haraphat01 :clap:
#### Good job so far :+1:  you have a few more requirements to fix before this project can be approved :point_down: 

**_1. Fix the horizontal scroll bar on the page._**

**_2. Fix the subscribe section and make it look like the original at each breakpoint. Importantly, the two images of the subscribe content should be positioned like the original_**

**_if you have any doubts or questions contact me on slack._**

### Reviewed by Temitayo Adeleke.

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
#### Status: Request changes

**Comment by haraphat01** [comment]
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

- There's no horizontal overflow on my side, but in case it shows over there, I have hidden it again. 

- I have also added the second picture on the subscription page. As I've discussed with you on slack, the subscribe button change sometimes as the page refreshes and likewise some elements on the original site, I had fixed before, I just changed it to this new one. 

I hope I'd get this project approved this time. Thank you

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @haraphat01, congratulations on a great job done :+1: However, we still have the horizontal scroll as identified by the previous TSE's :smile: A quick fix is to set the ```width:100%```. The image below shows how the app looks like on a small screen of  < 718px
fix

Kindly make this change and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by haraphat01** [comment]
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?
width

-  I have changed the width to [100%]

I hope I'd get this project approved this time. Thank you

**Review by serdg0 — CHANGES_REQUESTED** [review]
Hello @haraphat01 

Your project is **great**!, but we need to fix a thing previously adressed by a TSE. The horizontal scrollbar still appearing. I took a picture that might help in resolving this issue. There is a space where the navbar ends, and that last column ( which holds opinion, the debate, etc) is overflowing.

Selection_002

Hope that helps! 
Your project is looking awesome overall, so be patience! Remember this project will be watched by future potential employers! :moneybag: 

**Happy Coding!** :computer: :nerd_face:
**_Sergio Diaz_**

**Comment by haraphat01** [comment]
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

Attached is the screenshot of it looks now on my browser. 


width2

**Comment by sumancrest0001** [comment]
**STATUS: CHANGES REQUIRED
Hello @haraphat01** 

You have done a great job to fix the issues mentioned by the previous TSEs. You are very close to getting a green signal for this project. We have to fix a couple of minor issues. I believe it won't be that difficult considering your expertise.
[Required] Please make sure that the subscriber, newsletter, and footer section looks similar to the original one as we are doing cloning. currently, it looks a little bit off-color. For example, subscribers and newsletter sections should have equal height. Take a look :point_down: 
Screenshot from 2020-05-12 12-41-45
 It should be like  :point_down: 
Screenshot from 2020-05-12 12-50-59

**Comment by sumancrest0001** [comment]
please fix the alignment of the image and title of the more stories section on each screen size.
currently
Screenshot from 2020-05-12 12-42-15

**Comment by sumancrest0001** [comment]
On the small screen, the image and title of the subscribe section are overlapping. Fix this issue also.
Screenshot from 2020-05-12 12-43-46

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
After making the above-requested changes, send for another review.
Need help? Reach out to me on slack **Suman Shrestha**
Happy Coding !!!

**Comment by haraphat01** [comment]
This is my final work on my Bootstrap project. The instruction was to clone the www.newsweek.com, using only HTML, CSS, and Bootstrap, while ignoring any side that uses Javascript or contains sponsored posts, the aim of the project is to develop a fully responsive website that would fit on small, medium, large and, extra-large devices.
The technology used was basically;

Html

CSS and,

Bootstrap.

What have I done again?

- subscriber, newsletter, and footer section fixed

- alignment of the image fixed

- image and title of the subscribe section are overlapping also fixed.

I hope this is approved this time.

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

#### Great job!

#### Your project looks great!

### You can go ahead and merge this pull request!

### Happy coding! :computer:

## Lucas Mazo

**Pull request merged**